src/Provider/GraphHopper/Tests/GraphHopperTest.php

```diff
@@ -62,8 +63,11 @@ public function testGeocodeWithRealAddressAndLocale()
         }
 
         $provider = new GraphHopper($this->getHttpClient($_SERVER['GRAPHHOPPER_API_KEY']), $_SERVER['GRAPHHOPPER_API_KEY']);
-        $results = $provider->geocodeQuery(GeocodeQuery::create('242 Acklam Road, London, United Kingdom')->withLocale('fr'));
-
+        $results = $provider->geocodeQuery(
+            GeocodeQuery::create('242 Acklam Road, London, United Kingdom')
+                ->withLocale('fr')
+                ->withBounds(new Bounds(50, -10, 55, 10))
+        );
```

Comment:
Please use a new test to test this new "feature".

Reply:
done